feat: Document how to break out of the free trial (#2760)
Scalingo/documentation: src/_posts/platform/getting-started/2000-01-01-free-trial.md

### How to break out of the free trial?

You may want to leave the free trial period earlier to start using the platform to
its full potential. To do so, you need to enter a payment method on the
[web dashboard](https://dashboard.scalingo.com/billing). Once done, you can forcefully

> leave the trial period by clicking the `Stop free trial` button. Your billing will now
> start and limits will be gone.

Reviewer: Not sure but I think it's better that way

```suggestion
leave the trial period by clicking the `Stop free trial` button. Your billing now
starts and limits are gone.
```
